### ISSUE TITLE
js: ColumnType.Json serialization/deserialization, arbitrary object serialization

### ISSUE THREAD

**Issue by H-Plus-Time** [flatgeobuf/flatgeobuf#136]
At the moment, supplying a feature with compound-typed properties (arrays, objects) throws an error on serialize (and deserialize, if coming from an implementation with json support). My workaround has been to use typeof value === "object" in valueToType, and JSON.stringify and JSON.parse in buildFeature, parseProperties respectively.

Is this pretty much the direction the Json column type is heading in, similar to geobuf's handling of any non-primitive types (aside from datetimes)?

**Comment by bjornharrtell** [comment]
@H-Plus-Time yes the intention of the json column type is that it is simply a string but should (de)serialize into/from an object where possible (assuming the object can be represented as json). Looks like we could improve the TS implementation to do this by default.
